modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -51,15 +76,50 @@ See link:https://nmstate.io/examples.html#interfaces-ethernet[NMState] for addit
                 next-hop-interface: <NIC1_name>
 ----
 <1> Add NMState YAML syntax to configure host interfaces.
-
-[TIP]
++
+[NOTE]
 ====
-Consider saving the `networkConfig` YAML syntax to a file and testing it using the NMState command line interface before including it in the `install-config.yaml` file, because the installer will not check the NMState YAML syntax. Execute `nmstatectl gc <yaml-config>` to test the syntax. Errors in the YAML syntax might result in a failure to apply the network configuration. Additionally, maintaining the validated YAML syntax is useful when applying changes using Kubernetes NMState after deployment or when expanding the cluster.
+When configuring static IP addresses, be aware that while the cluster's nodes can be assigned a static IP address as part of the `install-config` file, the bootstrap VM cannot be assigned a static IP address using the 'install-config' file. 
```

Comment:
Do we need this note when there are also instructions below to assign a static ip for the bootstrap vm?

Reply:
@awolffredhat no, it can be removed if we're providing instructions later in the doc.